packages/react-jss/src/JssProvider.js

```diff
@@ -20,85 +20,88 @@ type Props = {
   generateId?: GenerateId,
   classNamePrefix?: string,
   disableStylesGeneration?: boolean,
+  sheetOptions: StyleSheetFactoryOptions,
   children: Node
 }
 
 export default class JssProvider extends Component<Props> {
   static propTypes = {
-    ...propTypes,
+    registry: PropTypes.instanceOf(SheetsRegistry),
+    jss: PropTypes.shape({}),
     generateId: PropTypes.func,
     classNamePrefix: PropTypes.string,
     disableStylesGeneration: PropTypes.bool,
-    children: PropTypes.node.isRequired
+    children: PropTypes.node.isRequired,
+    sheetOptions: PropTypes.shape({})
   }
 
-  static childContextTypes = contextTypes
+  managers: {} = {}
```

Comment:
I ment we could use the same here {[key: number]: SheetsManager} or even better reference a type Managers

Reply:
Done.